opentelekomcloud/resource_opentelekomcloud_cce_addon_v3.go

```diff
@@ -0,0 +1,229 @@
+package opentelekomcloud
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	golangsdk "github.com/opentelekomcloud/gophertelekomcloud"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/cce/v3/addons"
+)
+
+func resourceCCEAddonV3() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceCCEAddonV3Create,
+		Read:   resourceCCEAddonV3Read,
+		Update: resourceCCEAddonV3Update,
+		Delete: resourceCCEAddonV3Delete,
+
+		Schema: map[string]*schema.Schema{
+			"template_version": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"cluster_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"template_name": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Computed: true,
```

Comment:
I suppose that we need to validate this argument both with `template_name`
If it's possible